src/feedback_forensics/app/callbacks.py

```diff
@@ -390,8 +402,32 @@ def _get_default_annotator_cols_config(data) -> str:
             datasets[0], data
         )
         avail_datacol_annotator_names = _get_datacol_annotator_names(datasets[0], data)
```

Comment:
Can we also use the `get_annotators_by_type` function here? (by moving this down)

Reply:
Done (refactored this to avoid the datacol/principle/model split).